admin/admin_token.go

```diff
@@ -0,0 +1,64 @@
+package admin
+
+import (
+	"crypto/hmac"
+	"crypto/sha256"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/vocdoni/blind-csp/types"
+	"go.vocdoni.io/dvote/crypto/ethereum"
+)
+
+/**
```

Comment:
Please use the standard go doc format (https://tip.golang.org/doc/comment), this way it is automatically parsed by documentation tools.

Instead of /* use //, and do not leave a blank line at the end of the comment